runtime/compiler/optimizer/EscapeAnalysisTools.cpp

```diff
@@ -88,33 +84,35 @@ void TR_EscapeAnalysisTools::insertFakeEscapeForOSR(TR::Block *block, TR::Node *
       _comp->getOSRCompilationData()->printMap(induceDefiningMap);
       }
 
+   TR_BitVector symRefsToLoad(0, _comp->trMemory()->currentStackRegion(), growable);
+
    // Gather all live autos and pending pushes at this point for inlined methods in _loads
```

Comment:
Does this comment need to be updated to account for your change ?

Reply:
Corrected this comment in commit 353b298f097900d733cbd4f10a87f4ded688f3db.  When you're good with the changes, I can squash this commit down.